rps.js

```diff
@@ -0,0 +1,22 @@
+// JS Functions HW  , Q1 
+// Rock, Paper, Scissors (watch the game)
+
+function random(str) {
+	let randomNum = Math.floor(Math.random()*str.length) ;
+    return str[ randomNum ];
+}
+
+
+function playGame( userPicked ) {
+    let computerPicked = [ 'rock', 'paper', 'scissors' ];
+    let userScore = [ 'win', 'lose', 'tie' ];
+
+    let result = "Computer picked: "+ random(computerPicked);
```

Comment:
you need if conditions to compare winning conditions.
I got this output "Computer picked: scissors  User picked: rock  User Score tie"